crypto/ec/curve25519.c

```diff
@@ -12,18 +12,17 @@
 #include <openssl/sha.h>
 
 #if defined(X25519_ASM) \
-    || ( !defined(PEDANTIC) && \
-         !defined(__sparc__) && \
-         (defined(__SIZEOF_INT128__) && __SIZEOF_INT128__==16) )
+    || ( (defined(__SIZEOF_INT128__) && __SIZEOF_INT128__ == 16) \
+         && !defined(__sparc__) )
 /*
  * Base 2^51 implementation.
  */
 # define BASE_2_51_IMPLEMENTED
 
 typedef uint64_t fe51[5];
 # if !defined(X25519_ASM)
-typedef unsigned __int128 u128;
-# endif
+typedef __uint128_t u128;
+#endif
```

Comment:
Nit: needs a space between "#" and "endif"

Reply:
Oh! Will do [with [skip ci]] once appveyor passes...